evalml/preprocessing/utils.py

```diff
@@ -42,8 +42,6 @@ def load_data(path, index, target, n_rows=None, drop=None, verbose=True, **kwarg
         # target distribution
         print(target_distribution(y))
 
-    X = infer_feature_types(X)
```

Comment:
RE comment for demo datasets, are we returning WW for `load_data`? If not, we should update the docstring but I feel like we should 😁